Add unit tests for JavaUtils, Base64Util and SQLParserUtil

Braavos6 opened the pull request:
I've analysed your codebase and noticed that `com.alibaba.chaosblade.exec.plugin.jvm.script.java.JavaUtils`, `com.alibaba.chaosblade.exec.plugin.jvm.Base64Util` and `com.alibaba.chaosblade.exec.common.util.SQLParserUtil` is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important in a subsequent PR.

petersear commented on chaosblade-exec-plugin/chaosblade-exec-plugin-jvm/src/test/java/com/alibaba/chaosblade/exec/plugin/jvm/script/java/JavaUtilsTest.java:
```diff
@@ -0,0 +1,38 @@
+public class JavaUtilsTest {
+
+    @Test
+    public void testGetClassName() {
```
Add assert that checks for an actual class name rather than ""?

Braavos6 replied:
Done

petersear commented on chaosblade-exec-plugin/chaosblade-exec-plugin-jvm/src/test/java/com/alibaba/chaosblade/exec/plugin/jvm/script/java/JavaUtilsTest.java:
```diff
@@ -0,0 +1,38 @@
+    @Test
+    public void testIsAsciiAlpha() {
+        Assert.assertTrue(JavaUtils.isAsciiAlpha('B'));
+        Assert.assertTrue(JavaUtils.isAsciiAlpha('b'));
```
Add blank line between different assert types.

Braavos6 replied:
Done

petersear commented on chaosblade-exec-plugin/chaosblade-exec-plugin-jvm/src/test/java/com/alibaba/chaosblade/exec/plugin/jvm/Base64UtilTest.java:
```diff
@@ -0,0 +1,40 @@
+    }
+
+    @Test
+    public void testDecode() {
```
Can a test be added that doesn't return ""?

Braavos6 replied:
As we discussed this seemed a bit difficult

petersear reviewed (changes requested):
In the commit text, I think class for `Base64Util` should be `com.alibaba.chaosblade.exec.plugin.jvm.Base64Util`

Remember to replace `<fully-qualified classname>` in the PR text with the names of the classes tested.

petersear commented:
Also remember to update the title of the PR

petersear reviewed (changes requested):
The changes look good to me but the commit text still has an apache package in the list as pointed out previously. I think `org.apache.commons.codec.binary.Base64.Base64Util` should be `com.alibaba.chaosblade.exec.plugin.jvm.Base64Util`

Braavos6 committed:
Add unit tests for JavaUtils, Base64Util and SQLParserUtil

Fully-qualified classname
com.alibaba.chaosblade.exec.plugin.jvm.script.java.JavaUtils
com.alibaba.chaosblade.exec.plugin.jvm.Base64Util
com.alibaba.chaosblade.exec.common.util.SQLParserUtil

These tests were written using Diffblue Cover.

Braavos6 commented:
> The changes look good to me but the commit text still has an apache package in the list as pointed out previously. I think `org.apache.commons.codec.binary.Base64.Base64Util` should be `com.alibaba.chaosblade.exec.plugin.jvm.Base64Util`

Done
